### PR TITLE
Sign up

### PR DESCRIPTION
**Pull request by Jpropst**

### CODE REVIEW

**Comment by Jpropst** [comment]
Lab 1 Signin- Propst

File: sign-in/index.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    width: 400px;
+    margin: 100px auto;
+    text-align: center;
+    box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.16);
+}
+
+button {
+    background-color: #7bd9d5;
+    color: #ffffff;
+    border: 0px solid black;
```
Comment (KatSchultz) [inline sign-in/index.css:52]: Could use border: none; to make this a little shorter.

File: sign-in/index.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    margin-right: auto;
+    margin-left: auto;
+    position: relative;
+    background-color: #e4e4e4;
+    color: #e4e4e4 ;
+    border-color: #ee7779;
+    border-style: solid;
+    width: 400px;
+    padding: 50px;
+    border-width: 100px;
```
Comment (KatSchultz) [inline sign-in/index.css:24]: Could combine the 3 border properties into "border: solid 100px #ee7779;" to clean up your code.

File: sign-in/index.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    position: relative;
```
Comment (KatSchultz) [inline sign-in/index.css:25]: Duplicate property can be removed.

**Comment by KatSchultz** [comment]
You crushed this exercise! You did a great job duplicating the form. My comments are just nit-picks to try to clean up your code a little bit.

File: sign-in/index.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <div class="container">
+        <div class="center">       
+        <h4 class="title">Have an account? Sign in</h4>
+            <div class="login-box">
```
Comment (KatSchultz) [inline sign-in/index.html:13]: I don't think this div is doing anything and could be deleted.

**Review by KatSchultz — COMMENTED** [review]
You crushed this exercise! You did a great job duplicating the form. My comments are just nit-picks to try to clean up your code a bit.

File: sign-in/index.css [hunk 5]
```diff
@@ -0,0 +1,68 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap') ;
```
Comment (KatSchultz) [inline sign-in/index.css:1]: Nice job bringing in the font!

**Review by KatSchultz — APPROVED** [review]

**Pull request closed without merging**